package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.2",
   "private": true,
   "engines": {
-    "node": "^18.17.0"
+    "node": "^20.11.0"
```

Comment:
the reset of the repo is 20.17.0, probably use the latest v20 which is 20.18.0

Reply:
nodejs image on the pipeline has `20.11.x` version installed, and if I request `20.17`, pipeline fails. I'll update to use 20.18 elsewhere.